backend/src/worker_flow.rs

```diff
@@ -302,6 +350,16 @@ async fn has_failure_module<'c>(
     .map_err(|e| Error::InternalErr(format!("error during retrieval of has_failure_module: {e}")))
 }
 
+fn next_retry(retry: &Retry, status: &RetryStatus) -> Option<(u16, Duration)> {
+    const MAX_FAILURES: u16 = 1000;
+    const HOURS: Duration = Duration::from_secs(60 * 60);
```

Comment:
We should error if the flow is ill-defined when it is started rather than clipping acceptable values.